Makefile

```diff
@@ -0,0 +1,38 @@
+YARPC=yarpc
+TEST_DIR=tests/unit
+
+.PHONY: create-env
+create-env:
+	rm -rf env
+	python3 -m pip install --user virtualenv
+	python3 -m venv env
+	source env/bin/activate
+
+.PHONY: install
+install:
+	pip install --upgrade pip
+	pip install -r $(TEST_DIR)/utils/requirements.txt
+
+.PHONY: test
+test: .cleanCoverage
+	pytest $(TEST_DIR)
+
+.cleanCoverage:
+	@echo 'cleaning coverage files ...'
+	rm -f .coverage
+	rm -fr htmlcov/
+
+.PHONY: open-report
+open-report: .cleanCoverage
+	pytest --cov=$(YARPC) --cov-report=term-missing --cov-report=html
+	open -a "Google Chrome" htmlcov/index.html
+
+.PHONY: help
+help:
+	@echo '==================================== HELP  ===================================='
+	@echo 'create-env - creates a virtualenv'
+	@echo 'install - install all run pre requisites to run tests and coverage report'
+	@echo 'test - run unit tests on all Python files'
+	@echo 'open-report - check code coverage of all Python files'
+	@echo 'lint - run pylint and flake8 on all your Python files'
+	@echo ''
```

Comment:
There must be an entry for Travis (pytest + coverage), `ci-test`.
See current Travis configuration. It should be moved to makefile.

Reply:
Linting tools will be moved to makefile too, but it's out of scope of current pr

Reply:
What do you mean for `ci-test` I am not understanding the ask for Travis -> Makefile

Reply:
I can make another issue to tackle pylint and flake8

Reply:
We need similar entry: https://github.com/aio-libs/aioredis/blob/f080f0125bcd2a8e9f294957d2bc8769d99c653e/Makefile#L90-L93

Reply:
I will make that change

Reply:
Just look at existing Travis config. You need to move pytest to makefile

Reply:
One concern I will give you is that if we use the `make` command in the Travis configuration it might cause a problem for Windows users, I am an Apple fa, if that is okay with you I will do it.

Reply:
so it will actually be moving all of this:
```
pip install -r tests/utils/requirements.txt
pre-commit run --all-files
pip3 install .
pytest --cov --cov-report=xml
```
Under the make target `ci-test` and then passing in `make ci-test` to the `script` target of the travis config

Reply:
In my local machine works well and it is because it's a linex machine, I am sure it will fail on Windows because you cannot run `make` commands on Windows.
![image](https://user-images.githubusercontent.com/36493537/66459172-8284e780-ea42-11e9-8a21-bb9061ab224b.png)

Reply:
Are you sure there is no `make`?

Reply:
https://stackoverflow.com/questions/32127524/how-to-install-and-use-make-in-windows
Seems chocolatey has make